src/main/java/org/broadinstitute/hellbender/tools/spark/sv/FindSmallIndelRegions.java

```diff
@@ -0,0 +1,135 @@
+import org.apache.spark.broadcast.Broadcast;
+import org.broadinstitute.barclay.argparser.ArgumentCollection;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.hellbender.cmdline.programgroups.StructuralVariationSparkProgramGroup;
+import org.broadinstitute.hellbender.engine.spark.GATKSparkTool;
+import org.broadinstitute.hellbender.tools.spark.utils.IntHistogram;
+import org.broadinstitute.hellbender.utils.read.GATKRead;
+
+import java.util.*;
+
+import static org.broadinstitute.hellbender.tools.spark.sv.StructuralVariationDiscoveryArgumentCollection.FindBreakpointEvidenceSparkArgumentCollection;
+
+@CommandLineProgramProperties(summary="Find regions likely to contain small indels due to fragment length anomalies.",
+        oneLineSummary="Find regions containing small indels.",
+        programGroup=StructuralVariationSparkProgramGroup.class)
+public final class FindSmallIndelRegions extends GATKSparkTool {
+    private static final long serialVersionUID = 1L;
+
+    @ArgumentCollection
+    private static final FindBreakpointEvidenceSparkArgumentCollection params =
+                new FindBreakpointEvidenceSparkArgumentCollection();
+
+    private static final int EVIDENCE_SIZE_GUESS = 1000;
+
+    @Override
+    public boolean requiresReads()
+    {
+        return true;
+    }
+
+    @Override
+    protected void runTool( final JavaSparkContext ctx ) {
+
+        final JavaRDD<GATKRead> allReads = getUnfilteredReads();
+        final Set<Integer> crossContigIgnoreSet = Collections.emptySet();
+        final SAMFileHeader header = getHeaderForReads();
+        final int maxTrackedFragmentLength = params.maxTrackedFragmentLength;
+        final SVReadFilter filter = new SVReadFilter(params);
+        final ReadMetadata readMetadata =
+                new ReadMetadata(crossContigIgnoreSet, header, maxTrackedFragmentLength, allReads, filter);
+        final Broadcast<ReadMetadata> broadcastReadMetadata = ctx.broadcast(readMetadata);
+
+        final List<BreakpointEvidence> evidenceList =
+            allReads
+                .mapPartitions(readItr -> {
+                    final List<BreakpointEvidence> evList = new ArrayList<>(EVIDENCE_SIZE_GUESS);
+                    final Finder finder = new Finder(broadcastReadMetadata.value(), filter);
+                    while ( readItr.hasNext() ) {
+                        finder.test(readItr.next(), evList);
+                    }
+                    finder.checkHistograms(evList);
+                    return evList.iterator();
+                })
+                .collect();
+
+        for ( final BreakpointEvidence evidence : evidenceList ) {
+            System.out.println(evidence);
+        }
+    }
+
+    public static final class Finder {
+        private static final int BLOCK_SIZE = 250;
+        private static final int EVIDENCE_WEIGHT = 5;
+        private static final float KS_SIGNIFICANCE = 1.E-5f;
```

Comment:
The reasoning was:
We perform ~12million tests (3Gb genome / new window every 250b).  So the expected number of false positives is 120 (12E6 * 1E-5).  We're ringing the bell 13,680 times, so one might expect < 1% of our calls to be FP.  Seemed reasonable to me.
I think we could pretty easily do something far more sophisticated, however:  we could pretty easily bring all 12million window values back to the driver and calculate q-values, or do something Bayesian to try to discover the right parameters.
BTW, BLOCK_SIZE and EVIDENCE_WEIGHT are equally seat-of-the-pants.  I don't think BLOCK_SIZE is too sensitive a parameter, but I don't really know that.  EVIDENCE_WEIGHT ought to be calculated from truth data.